torch/csrc/DynamicTypes.cpp

```diff
@@ -69,7 +69,11 @@ at::Type* get_type(const std::string& name, bool is_cuda, bool is_sparse) {
 
 PyTypeObject* getPyTypeObject(const at::Storage& storage)
 {
-  auto it = attype_to_py_storage_type.find(&storage.pImpl()->type());
+  auto attype = get_type(
```

Comment:
use globalContext().getType() and then you don't need to convert the scalar type to string.